### DISCUSSION TITLE
when restart skywalking ，can not dial tcp port 12800

### DISCUSSION THREAD

**Discussion by eternalsilencer** [apache/skywalking#8347] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

![image](https://user-images.githubusercontent.com/51810383/147539722-96e2a96a-72f7-460e-9827-4482eadd7ccd.png)
i deploy skywalking use helm ,when i restart skywalking service ,the issue happen

### What you expected to happen

when i restart,it will work fine

### How to reproduce

just use service docker restart 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Do you check the log? You just posted a k8s pod status. We can't tell what is going on.

**Comment by eternalsilencer** [comment]
![image](https://user-images.githubusercontent.com/51810383/147541281-c9e3cfad-d3e9-486c-b27b-29fe7e4fa852.png)

**Reply by eternalsilencer** [reply]
i cant not find err in the log

**Reply by wu-sheng** [reply]
The INFO log is the key, you clean the storage, but no init-container runs anymore.

**Reply by eternalsilencer** [reply]
![image](https://user-images.githubusercontent.com/51810383/147543900-75337ecb-8eb7-45f4-885f-d76bc55d6f0a.png)
you mean the init-container ,but i see it work well

**Reply by wu-sheng** [reply]
From the logs, it doesn't work well. The indices and templates should be created automatically through that. 
In all these, you are not just rebooting, you manually changed something. ElasticSearch server or something.

The best I could provide is the above, because we never would know what a private env could be.
